.vscode/autogpt-workspace.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "name": ".github",
+      "path": "../.github"
+    },
+    {
+      "name": ".vscode",
+      "path": "../.vscode"
+    },
+    {
+      "name": "assets",
+      "path": "../assets"
+    },
+    {
+      "name": "autogpt",
+      "path": "../autogpt"
+    },
+    {
+      "name": "benchmark",
+      "path": "../benchmark"
+    },
+    {
+      "name": "docs",
+      "path": "../docs"
+    },
+    {
+      "name": "forge",
+      "path": "../forge"
+    },
+    {
+      "name": "frontend",
+      "path": "../frontend"
+    },
+    {
+      "name": "autogpt_server",
+      "path": "../rnd/autogpt_server"
+    },
+    {
+      "name": "root_folder",
+      "path": ".."
+    }
+  ],
+  "settings": {
+    "python.analysis.useLibraryCodeForTypes": true,
+    "python.analysis.autoImportCompletions": true,
+    "python.analysis.typeCheckingMode": "basic",
+    "python.analysis.completeFunctionParens": true,
+    "python.analysis.inlayHints.functionReturnTypes": true,
+    "python.analysis.inlayHints.variableTypes": true,
+    "python.analysis.autoFormatStrings": true,
+    "python.analysis.inlayHints.callArgumentNames": "partial",
+    "python.analysis.fixAll": [
+      "source.convertImportFormat",
+      "source.unusedImports"
+    ]
+  },
+  "extensions": {
+    "recommendations": [
+      "charliermarsh.ruff",
+      "dart-code.flutter",
+      "ms-python.black-formatter",
+      "ms-python.vscode-pylance",
+      "prisma.prisma",
+      "qwtel.sqlite-viewer",
+      "joshx.workspace-terminals"
```

Comment:
VSCode already lets you choose when opening a terminal. If we have *10 folders* by default I don't want it to open up 10 terminals on start-up, because there is no way I'll actually be working on all 10 folders at the same time.

Reply:
Oh lol there’s a setting to disable that

Reply:
Then what's the point of the extension? Afaics that's the only thing it does.

Reply:
Removed